x-pack/plugins/cloud_defend/server/plugin.test.ts

```diff
@@ -55,15 +55,13 @@ const createMockFleetStartContract = (): DeeplyMockedKeys<FleetStartContract> =>
       fromRequest: jest.fn(async (_) => createFleetAuthzMock()),
     },
     fleetSetupCompleted: jest.fn().mockResolvedValue(undefined),
-    // @ts-expect-error 2322
     agentService: createMockAgentService(),
-    // @ts-expect-error 2322
     packageService: createMockPackageService(),
     agentPolicyService: createMockAgentPolicyService(),
     registerExternalCallback: jest.fn((..._: ExternalCallback) => {}),
     packagePolicyService: createPackagePolicyServiceMock(),
     createArtifactsClient: jest.fn().mockReturnValue(createArtifactsClientMock()),
-  };
+  } as unknown as DeeplyMockedKeys<FleetStartContract>;
```

Comment:
Could we replace this `createMockFleetStartContract` with the newly created `createFleetStartContractMock` imported from `fleet/server/mocks`?

Reply:
Thanks for the feedback @opauloh . Just pushed the changes you suggested.